examples/interrupt-li/interrupt_li/interrupt.py

```diff
@@ -0,0 +1,145 @@
+# Copyright AGNTCY Contributors (https://github.com/agntcy)
+# SPDX-License-Identifier: Apache-2.0
+import asyncio
+from typing import Any
+
+from dotenv import load_dotenv
+from llama_index.core.agent.react import ReActChatFormatter, ReActOutputParser
+from llama_index.core.llms.llm import LLM
+from llama_index.core.memory import ChatMemoryBuffer
+from llama_index.core.tools.types import BaseTool
+from llama_index.core.workflow import (
+    Context,
+    Event,
+    HumanResponseEvent,
+    InputRequiredEvent,
+    StartEvent,
+    StopEvent,
+    Workflow,
+    step,
+)
+
+load_dotenv()
```

Comment:
Can you use `load_dotenv(dotenv_path=find_dotenv(usecwd=True))`. That would support `.env` files in the root of the repo for developers, I believe.